source/ReactLoadableSSRAddon.js

```diff
@@ -60,11 +60,12 @@ export default class ReactLoadableSSRAddon {
    * @returns {string} - Output path containing path + filename.
    */
   get manifestOutputPath() {
-    if (path.isAbsolute(this.options.filename)) {
-      return this.options.filename;
+    const filename = this.options.filename;
```

Comment:
Alternatively, 

```js
const { filename } = this.options;

Reply:
Looks good your fix, but could you please switch the implementation to the `destructuring assignment` one?
Just for the sake of code consistence :)